vulture_os/toolkit/api_parser/symantec/symantec.py

```diff
@@ -102,21 +105,26 @@ def test(self):
     def execute(self, token="none"):
         self.update_lock()
 
+        default_last_api_range_sec = 600  # get 10 minutes of logs
         last_api_call = self.last_api_call.replace(minute=0, second=0, microsecond=0)
         now = timezone.now().replace(minute=0, second=0, microsecond=0)
-        if last_api_call > now - datetime.timedelta(hours=1):
+        if last_api_call > now - datetime.timedelta(minutes=default_last_api_range_sec/60):
             return
 
         # If we have less than one hour
-        if (now - last_api_call).total_seconds() <= 60*60 and timezone.now().minute < 30:
+        if (now - last_api_call).total_seconds() <= default_last_api_range_sec and timezone.now().minute < (default_last_api_range_sec/60)/2:
```

Comment:
I think the second part of the condition was to avoid querying the API on the first half of an hour, that shouldn't have been changed (and the operation should have been commented to know what/why this was like this...)